meetup/test/test_model_relationship.py

```diff
@@ -0,0 +1,50 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+from meetup.models import Venue, Speaker, Profile, Program, ProgramCategory
+
+User = get_user_model()
+
+
+class ModelRelationshipTestCase(TestCase):
+    """This test class is for proving that the relationship among declared django models are organized successfully"""
+
+    def test_user_has_the_profile(self):
+        """This test is for proving that a user has a profile"""
```

Comment:
I think we can add attribute exist check.
you can use [model meta](https://docs.djangoproject.com/en/1.11/ref/models/meta/) before creating instance

Reply:
That means, verifing the models' attributes between model what I made on `models.py` and model what I declared in test?

Reply:
@NohSeho yes. IMHO declaration should be tested. In my case, I start to test from import

Reply:
I see, I will add that test code soon